CHANGELOG.md

```diff
@@ -35,6 +35,11 @@ release.
 
 ## [Unreleased]
 
+### Added
+
+- CSMCamera can now read and use the body rotation from ALE produced ISDs [#5072](https://github.com/DOI-USGS/ISIS3/pull/5072)
+- CSMSkyMap added to CSMCamera for use with local rover projections in ISIS [#5072](https://github.com/DOI-USGS/ISIS3/pull/5072)
```

Comment:
We don't normally allow more than one changelog but this is an old PR so we can probably let it slide.